### ISSUE TITLE
GPU ACCELERATE

### ISSUE THREAD

**Issue by lfxx** [jianboqi/CSF#45]
Hi,this repo has impressive performance.But on large pcd,it runs quite slow.Any plan to release a gpu accelerate version?

**Comment by jianboqi** [comment]
Yes, GPU is currently under consideration, but it is not implemented now.

**Comment by jianboqi** [comment]
A GPU version is implemented within our new GUI based point cloud processing software. You can refer to http://easypoint.xyz
